packages/dataviews/src/layout-types.ts

```diff
@@ -0,0 +1,20 @@
+export type LayoutType = 'regular' | 'panel' | 'card';
+export type LabelPosition = 'top' | 'side' | 'none';
+
+export type RegularLayout = {
+	type: 'regular';
+	labelPosition?: LabelPosition;
+};
+
+export type PanelLayout = {
+	type: 'panel';
+	labelPosition?: LabelPosition;
+};
+
+export type CardLayout = {
+	type: 'card';
+	labelPosition?: Extract< LabelPosition, 'top' | 'none' >;
+	opened?: boolean;
```

Comment:
`labelPosition`: I understand where this is coming from, but this is a boolean camouflaged as a set. The semantics of this prop are different from the semantics of `labelPosition` in regular and panel layouts, and so I think its name should reflect that. Some options: `withHeader`, `hasHeader`, etc.

`opened`: this is a small nit. `isOpened` would better communicate that is a boolean.